classes/clean.php

```diff
@@ -22,14 +22,15 @@
 
 namespace local_datacleaner;
 
-use core\base;
```

Comment:
This was probably an accident initially

Reply:
good point. I'll revert it back. Just in case